types/artifacts.d.ts

```diff
@@ -415,20 +415,24 @@ declare global {
         /** The raw height attribute of the image element. CSS images will be set to the empty string. */
         attributeHeight: string;
         /** The CSS width property of the image element. */
-        cssWidth?: string | undefined;
+        cssWidth: string | undefined;
         /** The CSS height property of the image element. */
-        cssHeight?: string | undefined;
+        cssHeight: string | undefined;
         /** The BoundingClientRect of the element. */
         clientRect: {
           top: number;
           bottom: number;
           left: number;
           right: number;
         };
+        /** The CSS position attribute of the element */
+        positioning: string;
```

Comment:
WDYT about...

- `position`
- `cssPosition`
- `computedStylePosition`
- `cssComputedPosition`
- `computedPosition`

trying to align on these properties matching with the DOM as much as possible and `positioning` feels a little out of line with that. I think my personal favorites might be `computedStylePosition` or `cssComputedPosition` since it isn't as easily confused for the position in the viewport/document and emphasizes it's not the raw CSS value but whatever gCS came up with.

Reply:
Changing it sounds good! I think I'll go with `cssComputedPosition` because it follows a similar structure to `cssWidth / cssHeight`